backend/lcfs/db/migrations/versions/2024-12-09-22-33_cd8698fe40e6.py

```diff
@@ -11,7 +11,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "cd8698fe40e6"
-down_revision = "9206124a098b"
+down_revision = "7ae38a8413ab"
```

Comment:
please double check this migration id is in the correct order.

Reply:
updating this another migration appeared before mine.